numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -1299,6 +1304,96 @@ PyUFunc_MixedDivisionTypeResolver(PyUFuncObject *ufunc,
                                         type_tup, out_dtypes);
 }
 
+#define NOT_CONTIGUOUS(a) (PyArray_NDIM(a) >=2 && !( \
```

Comment:
Could you add a comment with a link to the github issue?

Reply:
added